docs/configuration.rst

```diff
@@ -405,6 +405,61 @@ Use ``style_start`` and ``style_end`` like this::
       }
    ]
 
+.. _needs_filter_data:
+
+needs_filter_data
+~~~~~~~~~~~~~~~~~
+
+This option allows to custom define extra filter.
```

Comment:
This option allows to use custom data inside a filter-string. (filter-string should be linked to the related chapter)